dev/dockerfiles/devel/main.Dockerfile

```
@@ -402,7 +402,7 @@ export PROMPT_COMMAND=\"history -a; \$PROMPT_COMMAND\";\n\
  && apt autoremove -y && apt clean \
  && rm -rf /tmp/* /var/tmp/*
 
-ENV NODE_PATH=/usr/local/lib/node_modules
+ENV NODE_PATH=/usr/local/lib/node_modules:/opt/rapids/node/node_modules
```

Idk how docker treats `:`... maybe quoting this would work? 
```suggestion
ENV NODE_PATH="/usr/local/lib/node_modules:/opt/rapids/node/node_modules"
```

---

This does not seem to fix. It does appear that the jlab process does not get the original shell env vars, or at least very little show up in `os.environ` in a python nb